src/openeo_test_suite/tests/general/payload_examples/post_validation/post_validation_1.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
Does the test suite somehow makes sure that the processes used here are actually available?

Reply:
Not currently, how would you suggest that we separate testing for the existence of processes and this specific process. The included process includes "subtract", "multiply", "sum" and "divide". I was hoping we can just assume the existence of those very basic processes. 

If we cannot assume the existence of a single process then I am confused how I am supposed to construct a valid example for the validation endpoint at all.

Reply:
I'd just do the same as in process testing: skip if not available with a corresponding message.